resources/catalog/widgets.json

```diff
@@ -191,6 +191,12 @@
       "library": "widgets",
       "description": "A widget that attempts to size the child to a specific aspect ratio."
     },
+    {
+      "name": "Autocomplete",
+      "parent": "StatelessWidget",
+      "library": "material",
+      "description": "{@macro flutter.widgets.RawAutocomplete.RawAutocomplete}"
```

Comment:
Unrelated to this PR, but depending on how this data is intended to be used, would it be better if this was resolved to the underlying comment?